chia/rpc/wallet_rpc_api.py

```diff
@@ -1799,9 +1824,6 @@ async def did_find_lost_did(self, request) -> EndpointResult:
         p2_puzzle, recovery_list_hash, num_verification, singleton_struct, metadata = curried_args
 
         hint_list = compute_coin_hints(coin_spend)
-        old_inner_puzhash = DID_INNERPUZ_MOD.curry(
```

Comment:
why is removing this code relevant for memos?

Reply:
removed